clang/lib/DPCT/TypeLocRewriters.cpp

```diff
@@ -69,6 +81,31 @@ std::shared_ptr<TypeLocRewriterFactoryBase> createTypeLocRewriterFactory(
       std::forward<std::function<TypeNameT(const TypeLoc)>>(TypeNameCreator));
 }
 
+// Print a templated type. Pass a STR("") as a template argument for types with
+// no template argument e.g. MyType<>, if enable ctad, use a C++17 class
```

Comment:
empty template argument e.g. MyType<>. If --enable-ctad is set, the template arguments which could be deduced with class template argument deduction(CTAD) will be omitted in the generated code.

Reply:
done